docs/assembly.rst

```diff
@@ -699,7 +694,7 @@ The following assembly will be generated::
         mstore(ret, r)
         return(ret, 0x20)
       }
-      default: { jump(invalidJumpLabel) }
+      default: { revert(0, 0) }
```

Comment:
Oh that is weird, because it does not comply with the statement "if the invalid opcode can be reached, the contract has a bug". We should change that.